cs/firefox-ios.xliff

```diff
@@ -1797,7 +1801,6 @@
       </trans-unit>
       <trans-unit id="Logins.DevicePasscodeRequired.Message" xml:space="preserve">
         <source>To save and autofill logins and passwords, enable Face ID, Touch ID or a device passcode.</source>
-        <target>Pro ukládání a automatické vyplňování přihlašovacích údajů si na svém zařízení nastavte Face ID, Touch ID nebo kód zámku.</target>
```

Comment:
These translations should not be removed. My guess is that you did your export on an obsolete version of the repository (these strings were approved almost 2 weeks ago, Dec 8).

Please update the repository and redo the export